ls1intum/Artemis
# Fix assement due date display in Course Tab

### Checklist
- [X] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de (currently only on localhost).

### Motivation and Context


It fixes issue #1534 
### Description

I added the evaluation of assessmentDueDate to the method `convertDateArrayFromServer`, as it was missing there.
### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Create an exercise without Assessment Due Date
4. Assessment Due Date should not be displayed in exercise overview

This should work for all exercise types.
### Screenshots


![image](https://user-images.githubusercontent.com/50466283/83328869-84781100-a286-11ea-86a4-f665e6b4c556.png)
